Example/TrustlySDK/TrustlyLightBoxViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  TrustlyLightBoxViewController.swift
+//  TrustlySDK_Example
+//
+//  Created by Marcos Rivereto on 06/04/23.
+//  Copyright © 2023 CocoaPods. All rights reserved.
```

Comment:
I don't think this should be Copyrighted because it is under the MIT license yeah? And definitely not by CocoaPods, yeah? :)